static/terraboard.js

```diff
@@ -312,6 +324,27 @@ app.controller("tbSearchCtrl", ['$scope', '$http', '$location', '$routeParams',
         $scope.attrKey = undefined;
         $scope.attrVal = undefined;
         $scope.results = undefined;
-        $scope.doSearch(1);
+        $location.url('/search');
+    }
+
+    $scope.createPermalink = function(page) {
+        var params = {};
+        if ($scope.resType != "") {
+            params.type = $scope.resType;
+        }
+        if ($scope.resID != "") {
+            params.name = $scope.resID;
+        }
+        if ($scope.attrKey != "") {
+            params.key = $scope.attrKey;
+        }
+        if ($scope.attrVal != "") {
+            params.value = $scope.attrVal;
+        }
+        if (page != undefined) {
+            params.page = page;
+        }
+        var query = $.param(params);
+        $location.url('/search?'+query);
```

Comment:
On my screen, the fields are a bit on top of each other for some reason

![screenshot from 2017-08-01 20-10-54](https://user-images.githubusercontent.com/650430/28839945-969a994e-76f5-11e7-8e93-0215ea5aa02b.png)